Dockerfile

```diff
@@ -1,9 +1,9 @@
 FROM node:20-alpine
 
-RUN apk add --no-cache curl=8.9.0-r0 python3=3.12.3-r1 && \
-	npm install -g wikibase-cli@18.0.3
+RUN apk add --no-cache curl=8.9.0-r0
 
-COPY --chmod=755 ./transferbot.sh /usr/bin/transferbot
-COPY --chmod=755 ./mangle_data.py /usr/bin/mangle_data
+COPY . /pkg
+RUN npm i -g /pkg
+COPY --chmod=755 docker-entrypoint.sh /usr/bin/docker-entrypoint
 
-ENTRYPOINT ["transferbot"]
+ENTRYPOINT ["docker-entrypoint"]
```

Comment:
This likely breaks how the image is called from api, but I have the feeling it's not done correctly there anyways (explictly relying on `transferbot` being available.